### ISSUE TITLE
Master has `bits` CLBO for eirini

### ISSUE THREAD

**Issue by andreas-kupries** [cloudfoundry-incubator/kubecf#1085]
**Describe the bug**

Deploy commit 74773f1ce0ecae59f2b8bcd4db2380284c2072a9 (head of master as of the time of writing) for eirini SA (minikube 21G RAM). The `bits` pod has a persistent CLBO. From the logs it seems to be some kind of cert issue around the TLS certs not matching some other.

Counter to this, deploying the older commit 0e134b6824d705ba228d27d9fb419b1ad37cc4db mostly identical (20G RAM, i.e. 1G less) does not show these symptoms.

This should get the bisection started. Due to my focus on #914 and the time it takes to bring up a kubecf on minikube I forewent doing the full bisection myself.

**To Reproduce**

See above.

**Expected behavior**

No CLBO, working `bits` pod.

**Environment**

Minikube, operators, etc. a per the referenced commits (bazel config).

**Comment by mook-as** [comment]
Taking on the assumption that "doesn't work at all" is critical.

**Comment by mook-as** [comment]
> 2020-06-29T22:27:42.237Z        ERROR   csr-reconciler  quarkssecret/certificatesigningrequest_reconciler.go:158        Failed to remove completed CSR: could not delete csr 'kubecf-bits-service-ssl': certificatesigningrequests.certificates.k8s.io "kubecf-bits-service-ssl" is forbidden: User "system:serviceaccount:cf-operator:cf-operator" cannot delete resource "certificatesigningrequests" in API group "certificates.k8s.io" at the cluster scope

That sounds like we're not picking up the fix for https://github.com/cloudfoundry-incubator/quarks-operator/issues/952 correctly. Indeed:
<details><summary><code>cf-operator-cluster</code> cluster role</summary>

```
- apiGroups:
  - certificates.k8s.io
  resources:
  - certificatesigningrequests
  verbs:
  - create
  - get
  - list
  - update
  - watch
```
</details>
The `delete` permission is missing.

This will be fixed by #1049 (the cf-operator bump to 5.x).

**Comment by jandubois** [comment]
@mook-as But how can this be a recent regression? Looking at `git diff 0e134b6..74773f1` I don't see any changes that should affect this.

**Comment by andreas-kupries** [comment]
I believe I might know. Starting up the first kcf clusters from the head of master I ran into troubles with local changes getting mixed in, so I had to kill the cluster a few times. I did not bring the whole minikube down, just kubecf, so it might have left something behind, like this CSR.

In the switch to the older commit I brought minikube down, so everything was fully flushed.

So, this ticket might be noise coming out of partial cleanup.
My apologies.
